src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/tab.cljs

```diff
@@ -10,6 +11,7 @@
     [org.broadinstitute.firecloud-ui.common.table-utils :as table-utils]
     [org.broadinstitute.firecloud-ui.common.style :as style]
     [org.broadinstitute.firecloud-ui.endpoints :as endpoints]
+    [org.broadinstitute.firecloud-ui.utils :as utils :refer [access-token]]
```

Comment:
ggrrkgkkl!!!rl!!%@#@%S! I *meant* to alphabetize. "utils" is before "workspace". in my sleepiness I forgot about the extra "page" package. humph.